scripts/closeChrome.scpt

```diff
@@ -0,0 +1,32 @@
+-- Check if app is running
+on is_running(appName)
+	tell application "System Events" to (name of processes) contains appName
+end is_running
+
+on run argv
+	-- Check if chrome is running
+	set chromeRunning to is_running("Google Chrome")
+	-- Set URL to the once gived in arguments
+	set lookupUrl to item 1 of argv
+
+	-- Engine
+	if chromeRunning then
+		tell application "Google Chrome"
+			set i to 0
+			set j to 0
+			repeat with w in (windows)
+				set j to j + 1
+				repeat with t in (tabs of w)
+					set i to i + 1
+					if (t's URL as string) contains lookupUrl then
+						tell t to close
+						tell w to activate
+
+						set (active tab index of window j) to i
+						return
+					end if
+				end repeat
+			end repeat
+		end tell
+	end if
+end run
```

Comment:
is... this... appleScript??
Will this run only on macs? What about windows/linux users?

Reply:
Yes :O
I couldn't do this by javascript on `auth-server` 😢 , so the alternative was opening a PR here...
I don't even have any windows or linux environment configured to develop for those operating systems :(

Do you use linux? Want to help me on this one? :)

Reply:
I'll try doing this later in bash. Don't know if it'll work, though.

Reply:
Oh. Turns out `pkill --newest chrome` should do, IF the person didn't open any new tabs while logging in.